tests/test_addons.py

```diff
@@ -292,8 +294,60 @@ def test_not_has_permission_not_logged_in(self):
 
 
 class TestAddonFileViews(OsfTestCase):
-    pass
 
+    @mock.patch('website.addons.base.views.codecs.open')
```

Comment:
Could we avoid much ofl this mocking by creating a `TestCase` that sets the `MFR_CACHE_PATH` to some temporary directory that gets removed on `tearDownClass`?